torch/_dynamo/variables/torch.py

```diff
@@ -823,6 +834,9 @@ def call_function(
         from . import ConstantVariable, SymNodeVariable, TensorVariable
         from .builder import wrap_fx_proxy
 
+        if self.torch_function_override_enabled(tx, args, kwargs):
+            return dispatch_torch_function(tx, self, args, kwargs)
```

Comment:
It's not guaranteed that Dynamo will run torch_function on the same APIs as eager-mode PyTorch. get_overrides() is supposed to contain all APIs in eager-mode that torch_function can interpose on, but we don't have sufficient testing for it. Are you planning on making this better?

Reply:
It is actually, in the torch_function_override_enabled check, I check that the function is in `_get_overridable_functions`, the same set eager uses.

Reply:
I'm not convinced that all functions that are torch_function overrideable in eager-mode are in `_get_overridable_functions`, because it's easy for me to add a manual torch function override in C++ and for that to not show up in the list. I'm happy to call this a pre-existing problem with `_get_overridable_functions` but the end effect might be that some APIs are overridable in eager mode but not Dynamo

Reply:
So I'm happy to ship this PR without figuring this out, but my larger question is if there is a divergence and then if we care about it.

Reply:
Discussed offline: there is a divergence we think, and we will potentially handle this in the future via crossref mode asserting that func is in the overridable functions - this however is not a priority, but I will try this and see how bad the state of the set is.